model/clusters_mgmt/v1/aws_infrastructure_access_role_grant_state_type.model

```diff
@@ -24,4 +24,10 @@ enum AWSInfrastructureAccessRoleGrantState {
 
 	// The attempt to grant access role to user ARN failed.
 	Failed
+
+	// The access role grant is being deleted.
+	Deleting
+
+	// The access role grant has been removed.
+	Removed
```

Comment:
It is a bit weird that one is `Deleting` and the other `Removed`. Should probably be `Deleting` and `Deleted` or `Removing` and `Removed`.

Reply:
I actually want them to be different.
`Deleting` is an action initiated by the user - at the end the grant record will not exists.
`Removed` is a state that the corresponding object in AWS has been removed by SRE, and the record is still in the DB to let the user know it has been removed.
The user will have to perform a delete action to actually delete the `removed` record from the DB

Reply:
OK, please explain that in the documentation of each value.

Reply:
Done